src/app/component/ProductAttributeValue/ProductAttributeValue.style.scss

```diff
@@ -161,9 +162,12 @@
                 padding-left: 1rem;
                 padding-bottom: 0;
                 order: 1;
+                text-align: left;
 
                 ~ label {
                     margin: 0;
+                    min-width: 1.5rem;
+                    top: .3rem;
```

Comment:
This look like a random positioning.

Reply:
@alfredsgenkins  Combined with the baseline, we achieve the result that checkbox is aligned with text first line.